supernovacontroller/sequential/i2c.py

```diff
@@ -5,6 +5,62 @@
 
 
 class SupernovaI2CBlockingInterface:
+    """
+    The SupernovaI2CBlockingInterface class provides methods to interact with I2C devices.
+
+    This class includes three primary methods:
+
+    - write(target_static_address, register_address, data):
+        Writes data to the specified I2C target device.
+
+    - read(target_static_address, length):
+        Reads data from the specified I2C target device.
+
+    - read_from(target_static_address, register_address, length):
+        Reads data from the specified I2C target device.
```

Comment:
There is room to enhance this comment so it is not the same comment as the description of the `read` method. My suggestion below:

_Reads data from the specified I2C target device, indicating the address of the internal register from which the data is read._